Update Readme.md (#1)
DevExpress-Examples/wpf-docklayoutmanager-merge-ribbon-controls: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T304338)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->
# WPF Dock Layout Manager - Automatically Merge Ribbon Controls

> The following control support automatic bar merging:

Reviewer: ```suggestion
The following controls support automatic bar merging:
```

* [DXTabControl](https://docs.devexpress.com/WPF/7975/controls-and-libraries/layout-management/tab-control/fundamentals/dxtabcontrol)
* [Dock Layout Manager](https://docs.devexpress.com/WPF/6191/controls-and-libraries/layout-management/dock-windows)
  

> In this example, the [docklayoutmanager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager) component implements the tabbed MDI interface. The main application window and tabs contains ribbon controls.

Reviewer: ```suggestion
In this example, the [docklayoutmanager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager) component implements the tabbed MDI interface. The main application window and tabs contain ribbon controls.
```

> The example merges a pages, page groups, and items from child ribbons to the main ribbon control.

Reviewer: Can you update this line as follows?

`The example merges pages, page groups, and items from child ribbons to the main ribbon control.`

> The example illustrates pages, page groups and items merging from child ribbons to the main ribbon control.

Reviewer: Do you mean the following?

`The example illustrates how pages, page groups, and items are merged from child ribbons to the main ribbon control.`

Author: The example merges a pages, page groups, and items from child ribbons to the main ribbon control.

Author: ```suggestion
The example merges a pages, page groups, and items from child ribbons to the main ribbon control.
```
